app/src/main/java/com/skydoves/disneycompose/ui/details/PosterDetails.kt

```diff
@@ -97,14 +97,7 @@ fun PosterDetails(
             .padding(12.dp)
             .clickable(onClick = { pressOnBack() })
         )
-        ImageBalloonAnchor(
-          reference = image,
-          modifier = Modifier
-            .fillMaxWidth()
-            .aspectRatio(0.85f),
-          content = poster.name,
-          onClick = { balloon, anchor -> balloon.showAlignBottom(anchor) }
-        )
```

Comment:
I think just removing the previous feature is not a fundamental solution.
I think the `ImageBalloonAnchor` covers the whole layouts, so we can resolve this issue by changing the sequence of the layouts.

```kotlin
ImageBalloonAnchor(
          reference = image,
          modifier = Modifier
            .fillMaxWidth()
            .aspectRatio(0.85f),
          content = poster.name,
          onClick = { balloon, anchor -> balloon.showAlignBottom(anchor) }
)
Icon(
          imageVector = Icons.Filled.ArrowBack,
          tint = Color.White,
          contentDescription = null,
          modifier = Modifier
            .constrainAs(arrow) {
              top.linkTo(parent.top)
            }
            .padding(12.dp)
            .clickable(onClick = { pressOnBack() })
)
```

Reply:
Sorry I am still kind of learning how to use GitHub and contributing. I pushed the change, but saw you already made the change on another screen. My apologies.